### PR TITLE
Add scotlandCSS conf

### PR DESCRIPTION
**Pull request by lauragift21**

### CODE REVIEW

**Comment by nimzco** [comment]
Awesome, thank you! 👍

File: conferences/2018/css.json [hunk 1]
```diff
@@ -69,5 +69,13 @@
     "city": "Paris",
     "country": "France",
     "twitter": "@dotCSS"
+  },
+  {
+    "name": "scotlandCSS",
+    "url": "http://scotlandcss.com/",
```
Comment (nimzco) [inline conferences/2018/css.json]: Could you just remove the trailing `/`?

**Comment by nimzco** [comment]
Thank you, sorry for this weird rule. I might just remove it in the future...

**Comment by lauragift21** [comment]
Yeah it's fine. :) Happy to make the correction.

**Pull request merged**